.buildkite/llvm-apple-pipeline.yml

```
@@ -1,3 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  DOCKER_REGISTRY_SECRET_PATH: "secret/observability-team/ci/docker-registry/prod"
+  MAKEFILE: "go/llvm-apple"
+  FILES_TO_CHECK: ".buildkite/llvm-apple-pipeline.yml:.buildkite/scripts/llvm-apple:go/llvm-apple"
+  DOCKER_FILTER_REF: "*/*/golang-crossbuild:llvm-apple*"
+  BUILDX: "0"
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: ":linux: Build LLVM Apple / Ubuntu X86_64 - {{matrix.debianVersion}}"
+    key: "build-ubuntu-x86"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.source == "ui" || build.branch == "main" || build.branch =~ /[0-9]+\.[0-9]+/
+    command:
+      - ".buildkite/scripts/llvm-fpm/build.sh ${MAKEFILE} ${FILES_TO_CHECK} ${DOCKER_FILTER_REF}"
+      - ".buildkite/scripts/llvm-fpm/publish.sh ${MAKEFILE} ${FILES_TO_CHECK}"
+    notify:
+      - github_commit_status:
+          context: "Build LLVM Apple / Ubuntu X86_64"
+    env:
+      TAG_EXTENSION: "-debian{{matrix.debianVersion}}-amd64"
+      REPOSITORY: "${STAGING_IMAGE}"
+      DEBIAN_VERSION: "{{matrix.debianVersion}}"
+    agents:
+      provider: "gcp"
+      image: "${IMAGE_UBUNTU_X86_64}"
+    matrix:
+      setup:
+        debianVersion:
+          - "10"
+          - "11"
+          - "12"
+
+  - label: ":linux: Build LLVM Apple / Ubuntu ARM - {{matrix.debianVersion}}"
+    key: "build-ubuntu-arm"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.source == "ui" || build.branch == "main" || build.branch =~ /[0-9]+\.[0-9]+/
```

Should it be added to the regex of branch delimiters ? 

```suggestion
    if: build.env("BUILDKITE_PULL_REQUEST") != "false" || build.source == "ui" || build.branch == "main" || build.branch =~ /^[0-9]+\.[0-9]+$$/
```

Note, `$` must be escaped. Here we use it to check for specific tags: https://github.com/elastic/elastic-package/blob/c3716bc4826521ee747fe367bd35c169e08b2080/.buildkite/pipeline.yml#L66